pkgs/development/python-modules/django-picklefield/default.nix

```diff
@@ -11,6 +11,9 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ django ];
 
+  # The PyPi source is missing test files
+  doCheck = false;
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.